news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -305,7 +305,8 @@ protected function getMetaFields($objArticle)
 					/** @var UserModel $objAuthor */
 					if (($objAuthor = $objArticle->getRelated('author')) instanceof UserModel)
 					{
-						$return['author'] = $GLOBALS['TL_LANG']['MSC']['by'] . ' <span itemprop="author">' . $objAuthor->name . '</span>';
+						$return['author'] = $GLOBALS['TL_LANG']['MSC']['by'] . ' <span class="author">' . $objAuthor->name . '</span>';
```

Comment:
```suggestion
						$return['author'] = $GLOBALS['TL_LANG']['MSC']['by'] . ' ' . $objAuthor->name;
```

I don't think that we need the `<span>` at all then.

Reply:
I thought people were maybe using `itemprop="author"` as a CSS selector so I did not want to take away the possibility to still select that.

Reply:
I highly doubt that. And even if they did, this markup was never intended to be used for styling, so it is OK to remove it IMHO.